### ISSUE TITLE
Extract save state info from your plugins?

### ISSUE THREAD

**Issue by marianopeck** [maca88/datatables.plugins#12]
Hi Maca,

You plugins....are soooo cool that my users are not happy for "memorize" their changes only during their web sessions. I saw something about your RemoteStateSave, but I may want kind of implement that in my own app (because of some particular requirements like preferences etc). So...it would love so much if there could be a general API/function/extension whatsover, that allow me to ask to the table I have rendered the following info (or for each column, the following info):
- The order of the columns (or index in the table)
- The width of each column
- Wether each column has been pinned or not (if true, where... in left or side side..and in which order).

Basically...what I wonder is if I could get a list with that info, so that I can have all the info to persist user changes in my app/database and then re-use them later on and get back the "changes" of the user. 

Would that be possible?
